CalibCalorimetry/EcalTrivialCondModules/interface/EcalTrivialConditionRetriever.h

```diff
@@ -219,6 +224,12 @@ class EcalTrivialConditionRetriever : public edm::ESProducer,
   std::vector<double> energyUncertaintyParameters_;
   std::vector<double> energyCorrectionObjectSpecificParameters_;
 
+  double sim_pulse_shape_EB_thresh;
```

Comment:
use a trailing `_` to denote member variables

Reply:
done